### DISCUSSION TITLE
The discussion of ideas for project

### DISCUSSION THREAD

**Discussion by IvanIsak2000** [IvanIsak2000/cosmoterm#20] [Ideas]
That place for discussion about project:
- choose technology Today it blockchain p2p messenger)
- cryptography : sha256
- choose interface (GUI/ TUI)
- Now app not support video/ gid/ audoi files, only **text**
- etc

**Comment by IvanIsak2000** [comment]
@rohankishore are you get that mention?

**Comment by IvanIsak2000** [comment]
Currently concept:


As for the tui branch: I want to update the existing terminal interactions or add PyQT6 altogether, since I saw it as a powerful tool.

In the tui branch, I wanted to use the python promt toolkit to display a separate terminal, that is, on the left there will be an opportunity to enter messages and send them, and on the right there will be messages from others

But my dream is of course a full-fledged GUI interface, I think it will be very cool if you can think of something interesting

Ideally, I imagine it as applications similar to Telegram desktop: the user's contacts will be on the left, that is, there will be a nickname and a user's socket

**Comment by rohankishore** [comment]
Yep i got the mention

**Comment by rohankishore** [comment]
For GUI, I'll pick PyQt6 since its versatile

**Reply by IvanIsak2000** [reply]
Bro, it is so possible to create messenger via PyQT6? 



Like this one:
![image](https://github.com/IvanIsak2000/cosmoterm/assets/79650307/abcedc9b-9f9f-4699-a405-ff3ec46c3b49)

**Comment by rohankishore** [comment]
Like `this` one? Might be too hard (I'll not say the word impossible tho)
